tests/locale/keymap_or_locale_x11.pm

```diff
@@ -0,0 +1,38 @@
+# SUSE's openQA tests
+#
+# Copyright © 2018-2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Keyboard layout test in console and display manager after boot
+# Maintainer: Martin Loviska <mloviska@suse.com>
+
+use base "locale";
+use strict;
+use testapi;
+
+
+sub run {
+    my ($self) = @_;
+    # uncomment in case of different keyboard than us is used during installation ( feature not ready yet )
+    # my $expected   = get_var('INSTALL_KEYBOARD_LAYOUT','us');
+    my $expected       = 'us';
+    my %keystroke_list = (
+        us => '`1234567890-=~!@#$%^&*()_+',
+        fr => '²&é"(-è_çà)=~1234567890°+',
+        de => '1234567890ß°!"§$%&/()=?',
+        cz => ';+ěščřžýáíé=1234567890%'
+    );
```

Comment:
Maybe you can shift ``%keystroke_list`` to the package locale. It's basically shared between both tests ( console and x11 ).

Reply:
I thought about this, but we might want to have it more scalable and visible
but you are right - the hash can actually go to the lib function as well

Reply:
Ah, ok! But doesn't it fail on line https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/6634/files#diff-824c96311858d5ae7f03f7bb6a5c9e5cR30 ? When keystroke_list is defined in tests/locale/keymap_or_locale_x11.pm 
 only?

Reply:
My bad! The part was collapsed :/